docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -430,3 +429,17 @@ to `false`, and then save the integration.
 
 // Add Javascript and CSS for tabbed panels
 include::{tab-widgets}/code.asciidoc[]
+
+
+[discrete]
+[[fleet-ece-config]]
+== {fleet} is not working when enabled in an {ece} deployment
+
+When an {ece} deployment is set up with APM and {fleet}, even though the deployment operates as expected {fleet} cannot be configured. Agent log files may contain a message like the following:
+
+[source,sh]
+----
+WARN [transport] transport/tcp.go:52 DNS lookup failure "instance-000000 0000": lookup instance-0000000000 on 169.254.165.225:23: no such host
+----
+
+This is a known issue with the Elastic Stack pack version 7.14 shipped with {ece} version 2.10. To resolve the problem, link:{ece-ref}/ece-manage-elastic-stack.html[re-upload a fresh copy] of the version 7.14 Elastic Stack pack to overwrite the original one. This issue will also be addressed in later Elastic stack packs and {ece} versions.
```

Comment:
```suggestion
This is a known issue with the {stack} pack version 7.14 shipped with {ece} version 2.10. To resolve the problem, link:{ece-ref}/ece-manage-elastic-stack.html[re-upload a fresh copy] of the version 7.14 {stack} pack to overwrite the original one. This issue will also be addressed in later {stack} packs and {ece} versions.
```

Reply:
I'm assuming here that "pack" is an ECE-related term because we don't use "pack" in the stack docs (AFAIK).

Reply:
Yup, that's exactly right. ECE comes with "stack pack" archived bundles of the stack products.